__tests__/app.test.js

```diff
@@ -301,6 +301,28 @@ describe("PATCH /api/articles/:article_id", () => {
   });
 });
 
+describe("DELETE /api/comments/:comment_id", () => {
+  test("204: responds with a no content message when comment successfully deleted", () => {
+    return request(app).delete("/api/comments/3").expect(204);
```

Comment:
It does seem strange but just checking the 204 is exactly all you need - a delete endpoint will not send anything on body!